stylesheet.css

```diff
@@ -148,3 +148,94 @@ a {
   text-decoration: none;
   color: white;
 }
+
+.workedOn {
+  display: inline-block;
+  width: 375px;
+  min-height: 3461px;
+  background-image: url("image/portfolio-background-shapes-mobile.png");
+  background-repeat: no-repeat;
+  background-size: 375px 2617.21px;
+}
```

Comment:
- [ ] It is required that you style the main container of the projects which is in your case the `div` with a `classname` of `workedon` with `display:grid`. Kindly fix that to have your project approved.